src/main/java/uk/gov/hmcts/reform/sendletter/queue/QueueLetter.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sendletter.queue;
```

Comment:
basically it is a model. do you think it makes sense to push `queue` under `model`?